js/preset.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 (function () {
-  var BLUR_VALUE = 3;
-  var BRIGHTNESS_VALUE = 2;
+  var MIN_BLUR = 0;
+  var MAX_BLUR = 3;
+  var MIN_BRIGHTNESS = 1;
+  var MAX_BRIGHTNESS = 2;
+  var MIN_SATURATION = 0;
+  var MAX_SATURATION = 100;
   var BRIGHTNESS_CORRECTION = 1;
   var DEFAULT_PIN_POSITION = '455';
```

Comment:
```suggestion
  var DEFAULT_PIN_POSITION = 455;
  var DEFAULT_LINE_WIDTH = DEFAULT_PIN_POSITION;
```
`'455'` можно без кавычек. при сложении со строкой `'px'` будет строка